python-pysynphot/PKGBUILD

```diff
@@ -0,0 +1,76 @@
+# Maintainer: Astro Benzene <universebenzene at sina dot com>
+pkgbase=python-pysynphot
+_pyname=${pkgbase#python-}
+pkgname=("python-${_pyname}" "python2-${_pyname}" "python-${_pyname}-doc")
```

Comment:
In this case, you need to add a file called `package.list`
and list all the `pkgname` in it, eg. https://github.com/arch4edu/arch4edu/blob/master/python-av/package.list

Reply:
Ah, I see. Added the `package.list` file for this PKGBUILD.